hamocc/mo_ncout_hamocc.F90

```diff
@@ -1892,6 +1916,10 @@ subroutine hamoccvardef(iogrp,timeunits,calendar,cmpflg)
       call ncdefvar3d(LVL_PREFDIC(iogrp),cmpflg,'p',                              &
            &   'p_diclvl','Preformed DIC',' ','mol C m-3',2)
     endif
+    if (use_shelfsea_res_time) then
+      call ncdefvar3d(LVL_SHELFAGE(iogrp),cmpflg,'p',                           &
+           &   'shelfage_lvl','Residence time of shelf water',' ','d',2)
```

Comment:
See my comment above, I think this should be `'shelfagelvl'`

Reply:
Done.